docs/calico-with-docker/without-docker-networking/README.md

```diff
@@ -63,13 +63,26 @@ You should see output like this on each node
 
 If you are not running in the cloud you may skip this step and jump to step 4.
 
-If you are running in the cloud, you will need to first configure an IP Pool
-with the `ipip` and `nat-outgoing` options.
+### 3.1 AWS
+
+If all your endpoints are in the same subnet of your VPC, disable Source/Dest. checks on each instance.  You can disable this with the CLI, or right click the instance in the EC2 console and `Change Source/Dest. Check` from the `Networking` submenu.
+
+On a node with the AWS CLI installed, for each instance:
+
+    aws ec2 modify-instance-attribute --instance-id <INSTANCE_ID> --source-dest-check "{\"Value\": false}"
+
+And then configure the `nat-outgoing` option on either Calico node:
```

Comment:
Maybe this could be clearer about running regardless of setting the Source/Dest. Check in CLI or in the EC2 console, I'd be worried people might think it is just when running the CLI.

Something like: "After disabling Source/Dest. Checks, configure the `nat-outgoing` option on your Calico IP pool from any Calico node:"